requirements/edx-sandbox/base.in

```diff
@@ -15,11 +15,11 @@ numpy==1.6.2                        # Numeric array processing utilities; used b
 pyparsing==2.2.0                    # Python Parsing module
 scipy==0.14.0                       # Math, science, and engineering library
 sympy==0.7.1                        # Symbolic math library
+-e git+https://github.com/edx/openedx-calc.git@stv/tests#egg=calc==0.4
```

Comment:
@stvstnfrd  BTW, I tried this same build without the editable flag here: https://github.com/edx/edx-platform/pull/20279 and it also succeeded so I don't think you need the `-e` on this.

Reply:
Cool, thanks @feanil , I can make that change; I guess I was still cargo-culting..

I was actually hoping we'd be publishing the new package to PyPI.
Assuming so, I'll update again, presumably `openedx-calc`?

Reply:
yea, I think we can land your current PR and then do a follow up to get it publishing to PyPI.  Do you want to wait on that before landing this one?

Reply:
Cool, updated to remove `-e` flag.
Note: will squash prior to merge.

Reply:
And that's fine.. just let me know when it's ready to go on PyPI later.